vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow/src/main/java/com/vaadin/flow/component/spreadsheet/SpreadsheetTable.java

```diff
@@ -68,10 +68,6 @@ public SpreadsheetTable(Spreadsheet spreadsheet, Sheet sheet,
         this.sheet = sheet;
         this.fullTableRegion = fullTableRegion;
         popupButtons = new HashMap<CellReference, PopupButton>();
-
-        if (isTableSheetCurrentlyActive()) {
-            initPopupButtons();
-        }
```

Comment:
I understand now what caused the difference in behavior. 

Before, by the time the `registerTable` is called, all `PopupButton` instances of the table are available (for the active sheet), so it makes sense to perform some actions on them (like calling `PopupButton#setHeaderHidden`) and then call `super.registerTable(table)` afterward. 
 
By removing this code and calling `reload()` in the `registerTable` method, we delay creating the instances so that they won't be available by any subclass that overrides the method.

Could you explain why this code has been removed?